src/components/Admin/IncomingGuests.jsx

```diff
@@ -3,55 +3,64 @@ import axios from './../../api/AxiosNoqApi';
 import Panel from "../Common/Panel";
 
 
-
-
-
 export default function IncomingGuests() {
 
     const [incomingBookings, setIncomingBookings] = useState([])
 
     useEffect( () => {
         axios.get ('/api/host/bookings/incoming')
         .then ( (response) => {
-            console.log(response)
-            
             if (response.status === 200) {
                 setIncomingBookings(response?.data)
             }
-            
         })
         .catch((error) => {
             console.log("Error while fetching incoming bookings data.", error);
         });
     }, []);
 
     const handleCheckIn = (bookingId) => {
-        //TODO
-        axios.get(`/api/host/bookings/${bookingId}/checkin`)
-        .then((response )=> {
-            
-            if(response.status === 200) {
-                setIncomingBookings(prevBokings => 
-                    prevBokings.map(booking => 
-                        booking.id === bookingId
-                        ? {...booking, status: {...booking.status, description: 'Checked In'}} :booking
-                    )
-        
-                )
-            }
+        const currentDate = new Date().toLocaleDateString()
+        const currentTime = new Date().toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })
+        axios.patch(`/api/host/bookings/${bookingId}/checkin`)
+            .then((res) => {
+                if (res.status === 200) {
+                    
+                    setIncomingBookings(prevBookings =>
```

Comment:
We should not update the list with date and time. We don't have logging for state changes for bookings yet, but that should be backend functionality and we don't have to set these values in the frontend. When backend gets the input to change booking state to checked_in, backend should log the date and time some way (to be defined).